nexus/src/external_api/http_entrypoints.rs

```diff
@@ -4659,6 +4659,7 @@ async fn rack_view(
 }]
 async fn sled_list_uninitialized(
     rqctx: RequestContext<Arc<ServerContext>>,
+    _query_params: Query<PaginatedById>,
 ) -> Result<HttpResponseOk<ResultsPage<UninitializedSled>>, HttpError> {
     let apictx = rqctx.context();
     let handler = async {
```

Comment:
- I don't think we want to offer a `sort_by` query parameter that we ignore
- I'm fine ignoring the `limit` parameter, but perhaps we could use it below
- I think we want to produce an error if a caller specifies a page token

```suggestion
    query: Query<PaginationParams<EmptyScanParams, ()>>,
) -> Result<HttpResponseOk<ResultsPage<UninitializedSled>>, HttpError> {
    let pag_params = query.into_inner();
    if let dropshot::WhichPage::Next(last_seen) = &pag_params.page {
        return Err(Error::invalid_value(last_seen.clone(), "bad page token").into());
    };
    let apictx = rqctx.context();
    let handler = async {
```

Reply:
Thanks for the tip! Fixed in 06a385178.